{
  "active_lock_reason": null,
  "assignee": null,
  "assignees": [],
  "author_association": "MEMBER",
  "body": "<strike>This unit test fails consistently when using go 1.13. As we will plan to upgrade to go 1.13 in the future, we should no doubt fix it:</strike>\r\n\r\nThe unit test currently has a workaround set in the Makefile, of `export GODEBUG=tls13=0`. This is require for the unit tests to not produce the following error.  According to the [go docs](https://golang.org/doc/go1.13#tls_1_3), this workaround will be removed in go 1.14.\r\n\r\n```\r\nok  \tvitess.io/vitess/go/memcache\t0.002s\r\nE1208 20:51:09.808962    6454 server.go:320] Cannot read post-SSL client handshake response from client 1 ([::1]:36288): tls: client didn't provide a certificate\r\nio.ReadFull(header size) failed\r\nW1208 20:51:09.809726    6454 auth_server_static.go:195] Config parsed using legacy configuration. Please update to the latest format: {\"user\":[{\"Password\": \"xxx\"}, ...]}\r\nW1208 20:51:10.991662    6454 server.go:363] Error authenticating user using MySQL native password: Access denied for user 'user1' (errno 1045) (sqlstate 28000)\r\nW1208 20:51:10.992049    6454 server.go:363] Error authenticating user using MySQL native password: Access denied for user 'user1' (errno 1045) (sqlstate 28000)\r\nW1208 20:51:11.005966    6454 server.go:451] Slow connection from client 1 (127.0.0.1:41664): 446.373\u00b5s\r\nE1208 20:51:11.014685    6454 server.go:273] mysql_server caught panic:\r\ntest panic attack!\r\n/home/morgo/vitess/go/mysql/server_test.go:93 (0x8a5e97)\r\n\tio/vitess/go/mysql.(*testHandler).ComQuery: panic(\"test panic attack!\")\r\n/home/morgo/vitess/go/mysql/conn.go:1070 (0x8465f7)\r\n\tio/vitess/go/mysql.(*Conn).execQuery: err := handler.ComQuery(c, query, func(qr *sqltypes.Result) error {\r\n/home/morgo/vitess/go/mysql/conn.go:771 (0x845e6d)\r\n\tio/vitess/go/mysql.(*Conn).handleNextCommand: if err := c.execQuery(sql, handler, more); err != nil {\r\n/home/morgo/vitess/go/mysql/server.go:458 (0x863dfd)\r\n\tio/vitess/go/mysql.(*Listener).handle: err := c.handleNextCommand(l.handler)\r\n/usr/local/go/src/runtime/asm_amd64.s:1357 (0x460c00)\r\n\tgoexit: BYTE\t$0x90\t// NOP\r\nE1208 20:51:11.034205    6454 conn.go:1117] Error in the middle of a stream to client 5 (127.0.0.1:41672): forced error after send (errno 1047) (sqlstate 08S01)\r\nW1208 20:51:11.063186    6454 server.go:395] Error authenticating user using MySQL native password: Access denied for user 'user1' (errno 1045) (sqlstate 28000)\r\nW1208 20:51:11.068802    6454 server.go:395] Error authenticating user using MySQL native password: Access denied for user 'user2' (errno 1045) (sqlstate 28000)\r\n--- FAIL: TestTLSServer (0.23s)\r\n    server_test.go:1043: No count found for version TLS12\r\nFAIL\r\nFAIL\tvitess.io/vitess/go/mysql\t2.349s\r\nok  \tvitess.io/vitess/go/mysql/ldapauthserver\t0.003s\r\n```",
  "closed_at": "2020-05-15T17:21:00Z",
  "closed_by": {
    "avatar_url": "https://avatars1.githubusercontent.com/u/388311?v=4",
    "events_url": "https://api.github.com/users/deepthi/events{/privacy}",
    "followers_url": "https://api.github.com/users/deepthi/followers",
    "following_url": "https://api.github.com/users/deepthi/following{/other_user}",
    "gists_url": "https://api.github.com/users/deepthi/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/deepthi",
    "id": 388311,
    "login": "deepthi",
    "node_id": "MDQ6VXNlcjM4ODMxMQ==",
    "organizations_url": "https://api.github.com/users/deepthi/orgs",
    "received_events_url": "https://api.github.com/users/deepthi/received_events",
    "repos_url": "https://api.github.com/users/deepthi/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/deepthi/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/deepthi/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/deepthi"
  },
  "comments": 2,
  "comments_url": "https://api.github.com/repos/vitessio/vitess/issues/5533/comments",
  "created_at": "2019-12-08T20:53:11Z",
  "events_url": "https://api.github.com/repos/vitessio/vitess/issues/5533/events",
  "html_url": "https://github.com/vitessio/vitess/issues/5533",
  "id": 534599087,
  "labels": [
    {
      "color": "d11f28",
      "default": false,
      "description": "",
      "id": 760789208,
      "name": "Type: Bug",
      "node_id": "MDU6TGFiZWw3NjA3ODkyMDg=",
      "url": "https://api.github.com/repos/vitessio/vitess/labels/Type:%20Bug"
    }
  ],
  "labels_url": "https://api.github.com/repos/vitessio/vitess/issues/5533/labels{/name}",
  "locked": false,
  "milestone": null,
  "node_id": "MDU6SXNzdWU1MzQ1OTkwODc=",
  "number": 5533,
  "performed_via_github_app": null,
  "repository_url": "https://api.github.com/repos/vitessio/vitess",
  "state": "closed",
  "title": "test suite requires deprecated GODEBUG=tls13=0 to pass",
  "updated_at": "2020-05-15T17:21:00Z",
  "url": "https://api.github.com/repos/vitessio/vitess/issues/5533",
  "user": {
    "avatar_url": "https://avatars2.githubusercontent.com/u/57982?v=4",
    "events_url": "https://api.github.com/users/morgo/events{/privacy}",
    "followers_url": "https://api.github.com/users/morgo/followers",
    "following_url": "https://api.github.com/users/morgo/following{/other_user}",
    "gists_url": "https://api.github.com/users/morgo/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/morgo",
    "id": 57982,
    "login": "morgo",
    "node_id": "MDQ6VXNlcjU3OTgy",
    "organizations_url": "https://api.github.com/users/morgo/orgs",
    "received_events_url": "https://api.github.com/users/morgo/received_events",
    "repos_url": "https://api.github.com/users/morgo/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/morgo/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/morgo/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/morgo"
  }
}